sdk/src/Core/Amazon.Runtime/Internal/Transform/UnmarshallerContext.cs

```diff
@@ -352,6 +352,8 @@ public class XmlUnmarshallerContext : UnmarshallerContext
         private bool disposed = false;
         private bool currentlyProcessingEmptyElement;
         private bool processEmptyElements = false; //Flip to true in v4
+        private string valueXmlName = "";
+        private string keyXmlName = "";
```

Comment:
Should the defaults be `key` and `value`?

Reply:
added defaults